internal/services/search/search_service_resource.go

```diff
@@ -161,6 +161,17 @@ func resourceSearchService() *pluginsdk.Resource {
 				Default:  true,
 			},
 
+			"semantic_search_sku": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  services.SearchSemanticSearchDisabled,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(services.SearchSemanticSearchDisabled),
```

Comment:
We should treat `disabled` as [a None value](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/schema-design-considerations.md#the-none-value-or-similar) - meaning we can remove this default and the possible value here:

```suggestion
				ValidateFunc: validation.StringInSlice([]string{
```

In favour of this field being empty/unset when null

Reply:
Fixed.